src/github.com/kelseyhightower/confd/backends/rancher/client.go

```diff
@@ -0,0 +1,135 @@
+package rancher
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"reflect"
+	"strconv"
+	"strings"
+	"time"
+
+	log "github.com/kelseyhightower/confd/log"
+)
+
+const (
+	MetaDataURL = "http://rancher-metadata/2015-07-25"
+)
+
+type Client struct {
+	Url        string
+	HttpClient *http.Client
+}
+
+func NewRancherClient(backendNodes []string) (*Client, error) {
+	url := MetaDataURL
+
+	if len(backendNodes) > 0 {
+		url = strings.Join([]string{"http://", backendNodes[0], "/2015-07-25"}, "")
+	}
+
+	log.Info(strings.Join([]string{"Using Rancher Metadata URL: ", url}, ""))
```

Comment:
Just do `log.Info("Using Rancher Metadata URL: ", url)` or `log.Infof("Using Rancher Metadata URL: %s", url)` depending on the logging API